developer-docs-site/docs/tutorials/your-first-nft-sdk.md

```diff
@@ -0,0 +1,376 @@
+        "map": {
+            "data": []
+        }
+    },
+    "description": "Alice's simple token",
+    "largest_property_version": "0",
+    "maximum": "1",
+    "mutability_config": {
+        "description": false,
+        "maximum": false,
+        "properties": false,
+        "royalty": false,
+        "uri": false
+    },
+    "name": "Alice's first token",
+    "royalty": {
+        "payee_address": "0x9df0f527f3a0b445e4d5c320cfa269cdefafc7cd1ed17ffce4b3fd485b17aafb",
+        "royalty_points_denominator": "1000000",
+        "royalty_points_numerator": "0"
+    },
+    "supply": "1",
+    "uri": "https://aptos.dev/img/nyan.jpeg"
+}
+
+=== Transferring the token to Bob ===
+Alice's token balance: 0
+Bob's token balance: 1
+
+=== Transferring the token back to Alice using MultiAgent ===
+Alice's token balance: 1
+Bob's token balance: 0
+```
+
+This example demonstrates:
+
+* Initializing the REST and Faucet clients
+* The creation of two accounts: Alice and Bob
+* The funding and creation of Alice and Bob's accounts
+* The creation of a collection and a token using Alice's account
+* Alice offering a token and Bob claiming it
+* Bob unilaterally sending the token to Alice via a multiagent transaction
+
+## Step 4: The SDK in Depth
+
+### Step 4.1: Initializing the Clients
+
+In the first step, the example initializes both the REST and Faucet clients. The REST client interacts with the REST API, whereas the Faucet client is a devnet
+ service for creating and funding accounts.
+
+<Tabs>
+  <TabItem value="python" label="Python">
+
+```python
+:!: static/sdks/python/examples/simple-nft.py section_1
+```
+
+`common.py` initializes these values as such:
+```python
+:!: static/sdks/python/examples/common.py section_1
+```
+  </TabItem>
+  <TabItem value="rust" label="Rust">
+
+In progress
+  </TabItem>
+  <TabItem value="typescript" label="Typescript">
+
+In progress
+  </TabItem>
+</Tabs>
+
+:::tip
+
+The URLs for both services, by default, point to our devnet services, however, they can be configured via setting following environmental variables: `APTOS_NOD
+E_URL` and `APTOS_FAUCET_URL`.
+
+:::
+
+
+### Step 4.2: Creating local accounts
+
+The next step is to create two accounts from the locally. [Accounts][account_basics] represent both on and off-chain state. The off-chain state consists of an
+address and the public, private key pair used to authenticate ownership. This step demonstrates how to generate that off-chain state.
+
+<Tabs>
+  <TabItem value="python" label="Python">
+
+```python
+:!: static/sdks/python/examples/simple-nft.py section_2
+```
+  </TabItem>
+  <TabItem value="rust" label="Rust">
+
+In progress
+  </TabItem>
+  <TabItem value="typescript" label="Typescript">
+
+In progress
+  </TabItem>
+</Tabs>
+
+### Step 4.3: Creating blockchain accounts
+
+In Aptos, each account must have an on-chain representation in order to support receive tokens, coins, and interacting in other DApps. In Aptos, an account rep
```

Comment:
`In Aptos, .... In Aptos, ...`

Let's make this 
`In Aptos, .. An account represents...`. Also, please don't cut the words off mid line.

The first sentence reads strange should be the same structure and tense for all 3 commas.  Plus `in order to support receive` doesn't read well.
```
In aptos, each account must have an on-chain representation in order to receive tokens, receive coins, and interact with other Dapps
```
I think it's supposed to be `dApps`?